gallery/src/components/NxDropdown/NxDropdownLinksExample.tsx

```diff
@@ -10,26 +10,24 @@ import { faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons';
 import { NxDropdown, NxFontAwesomeIcon, useToggle } from '@sonatype/react-shared-components';
 
 function NxDropdownActionsExample() {
-  const [isOpen, onToggleCollapse] = useToggle(false),
-      deleteFn = () => { alert('delete'); },
-      onClick = () => { alert('click'); };
+  const [isOpen, onToggleCollapse] = useToggle(false);
 
   return (
     <NxDropdown label="Links!"
                 className="extra-class"
                 isOpen={isOpen}
                 onToggleCollapse={onToggleCollapse}>
-      <a className="nx-dropdown-link disabled">
+      <a href="#/" className="nx-dropdown-link disabled">
```

Comment:
Plugin enforces that links are actual links. A lot of our examples were a bit lazy on this point and needed updating.

Note also that the plugin doesn't consider `#` to be good enough as a URL.  This is one of the plugin authors opinions that I find contentious.  I'm familiar with the bootstrap-style code where `href="#"` is used as an inaccessible cop out, which is what the plugin is trying to block, but there are other ways to use it and that's not how we were using it here.  At any rate though, it was easy to work around as our ui router treats `#` and `#/` the same.